local/README.md

```diff
@@ -0,0 +1,5 @@
+# Local
+
+[![GoDoc](https://godoc.org/github.com/golang/gddo?status.svg)](https://godoc.org/github.com/ubclaunchpad/inertia/local)
+
+This package contains utility functions that interact with a local git repository and creates config files for Inertia upon first-time setup.
```

Comment:
> creates config files for Inertia upon first-time setup.

it also handles all resource fetching the command line app - see [`doc.go`](https://github.com/ubclaunchpad/inertia/blob/master/local/doc.go) for the official summary